lib/chef/knife/ec_backup.rb

```diff
@@ -173,10 +173,11 @@ def chef_fs_paths(pattern_str, chef_fs_config, exclude=[])
         pattern = Chef::ChefFS::FilePattern.new(pattern_str)
         list = Chef::ChefFS::FileSystem.list(chef_fs_config.chef_fs, pattern)
         list = list.select { |entry| ! exclude.include?(entry.name) } if ! exclude.empty?
-        list.map {|entry| entry.path }
+        list.map {|entry| entry.path=~/\.json\z/ ? entry.path : entry.path<<'.json' }
```

Comment:
I know this seems minor, but I would consider breaking out this operation of maybe adding .json into a function like `normalize_path_name` or something rather than having this terse one-liner.